ci-operator/step-registry/cucushift/upgrade/check/capability/cucushift-upgrade-check-capability-commands.sh

```diff
@@ -386,11 +391,14 @@ expected_enabled_caps=""
 expected_implicit_caps=""
 expected_disabled_caps=""
 
-source_baseline_caps=$(get_caps_for_version_capset "${version_set[-1]}" "${baselinecaps_from_cluster}")
-target_baseline_caps=$(get_caps_for_version_capset "${version_set[0]}" "${baselinecaps_from_cluster}")
+#use caps_ref_version is the doc version to refer the capabilities as "MachineAPI is not part v4.13 or lesser, but same is applicable to all version as per doc 4.14 or later"
+caps_ref_version="${version_set[0]}"
+
+source_baseline_caps=$(get_caps_for_version_capset "${version_set[-1]}" "${baselinecaps_from_cluster}" "${caps_ref_version}")
```

Comment:
The problem is not from get_caps_for_version_capset, but `expected_implicit_caps` list is not correct, right? Currently `MachineAPI Build DeploymentConfig ImageRegistry` are regarded as new_caps after comparing source and target known caps, but actually `MachineAPI` is also in `target_baseline_caps`, so I think it should be stripped in `get_transfered_new_caps` func to get a correct `expected_implicit_caps` list.